engine/table/src/main/java/io/deephaven/engine/table/impl/lang/QueryLanguageFunctionGenerator.java

```diff
@@ -320,64 +34,60 @@ public static void main(String args[]) {
+                "        // noinspection SimplifiableBooleanExpression\n" +
+                "        return obj1 == obj2 || (!(obj1 == null ^ obj2 == null) && obj1.equals(obj2));\n" +
                 "    }\n" +
                 "    \n" +
                 "    @SuppressWarnings({\"unchecked\"})\n" +
                 "    public static int compareTo(Comparable obj1, Comparable obj2) {\n" +
-                "        if (obj1==null){\n" +
-                "            return (obj2==null) ? 0 : -1;\n" +
+                "        if (obj1 == null) {\n" +
+                "            return (obj2 == null) ? 0 : -1;\n" +
                 "        }\n" +
                 "        \n" +
-                "        if (obj2==null){\n" +
+                "        if (obj2 == null) {\n" +
                 "            return 1;\n" +
                 "        }\n" +
                 "\n" +
                 "        return obj1.compareTo(obj2);\n" +
                 "    }\n" +
                 "\n" +
-                "    public static boolean not(boolean a){\n" +
+                "    public static boolean not(boolean a) {\n" +
                 "        return !a;\n" +
                 "    }\n" +
                 "\n" +
-                "    public static Boolean not(Boolean a){\n" +
-                "        return a==QueryConstants.NULL_BOOLEAN ? QueryConstants.NULL_BOOLEAN : Boolean.valueOf(!a);\n" +
+                "    public static Boolean not(Boolean a) {\n" +
+                "        return a == QueryConstants.NULL_BOOLEAN ? QueryConstants.NULL_BOOLEAN : Boolean.valueOf(!a);\n"
+                +
                 "    }\n\n");
 
-        // ------------------------------------------------------------------------------------------------------------------------------------------------------------------
-        /* Now start the test class: */
-
-        testBuf.append(String.join("\n",
-                "//",
-                "// Copyright (c) 2016-202 Deephaven Data Labs and Patent Pending",
-                "//",
-                "// ****** AUTO-GENERATED CLASS - DO NOT EDIT MANUALLY",
-                "// ****** Run " + QueryLanguageFunctionGenerator.class.getSimpleName() + " to regenerate",
-                "//",
-                "// @formatter:off",
-                ""));
-
-        testBuf.append("package io.deephaven.engine.table.impl.lang;\n\n");
-
-        testBuf.append("import io.deephaven.util.QueryConstants;\n\n");
-        testBuf.append("import junit.framework.TestCase;\n\n");
-
-        testBuf.append("@SuppressWarnings({\"unused\", \"WeakerAccess\", \"NumericOverflow\"})\n");
-        testBuf.append("public final class TestLanguageFunctionUtil extends TestCase {\n\n");
-
         // ------------------------------------------------------------------------------------------------------------------------------------------------------------------
 
+        // Class<?>[] classes =
```

Comment:
Why the new commented dead code?

Reply:
This is to generate the functions as near as possible to the expected order (so the diff isn't terrible).  I want to uncomment this and produce all datatypes together once we are happy with the signatures.